tests/test_fieldtypes.py

```diff
@@ -689,7 +690,9 @@ def test_path() -> None:
         ),
     ],
 )
-def test_path_multiple_parts(path_parts, expected_instance) -> None:
+def test_path_multiple_parts(
+    path_parts: Tuple[str | pathlib.PurePath, ...], expected_instance: Type[pathlib.PurePath]
```

Comment:
```suggestion
    path_parts: Tuple[str | pathlib.PurePath, ...], expected_instance: type[pathlib.PurePath]
```
This is legal these days (unless it's another thing not allowed by 3.8, in which case ignore this comment).

Reply:
I thought this got introduced in python 3.9 but the python compatibility test passes with `type` instead of `Type`